src/app/main/service/habit-assign/habit-assign.service.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { HabitAssignService } from './habit-assign.service';
+import { habitAssignLink } from '../../links';
+import {
+  LISTOFHABITS,
+  NEWHABIT,
+  ASSIGNRESPONSE,
+  MODIFIEDASSIGNRESPONCE,
+  HABITSFORDATE,
+  HABITSWITHTHESAMEHABITID
+} from '../../mocks/habit-assign-mock';
+
+describe('HabitService', () => {
+  let service: HabitAssignService;
+  let httpMock: HttpTestingController;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [HabitAssignService],
+      imports: [HttpClientTestingModule]
+    });
+    httpMock = TestBed.get(HttpTestingController);
+    service = TestBed.get(HabitAssignService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
```

Comment:
let's add an empty space before each it... so it will be easier to read it

Reply:
Empty space was added